### PR TITLE
Update BattleShip Project page

### PR DESCRIPTION
**Pull request by memcmahon**
Broke project into iterations and added new-style rubric.

### CODE REVIEW

**Comment by BrianZanti** [comment]
I would separate the iterations into separate pages, and add a note on the main "requirements" page that says they should complete one iteration before moving to the next. The reason being that since we are demoing the projects, it would be great if everyone gets something for the demo

**Comment by BrianZanti** [comment]
Add a section describing their check in and evaluation (the demo). Give them a guide as to what they need to do in the demo

File: module1/projects/battleship/requirements.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+6. [Computer Shot Sequence](#computer-shot-sequence)
+
+### Iteration 3
+
+7. [Ship Hit Sequence](#ship-hit-sequence)
+8. [End Game Sequence](#end-game-sequence)
+
+### Iteration 4
+
+9. Make your game playable through a web browser. You may find [This Tutorial](/module1/projects/http_tutorial) helpful.
```
Comment (BrianZanti) [inline module1/projects/battleship/requirements.md]: Unfortunately, students probably won't be willing to tackle this iteration 4 😞 Maybe for I4 we could say that when starting the game, the user should be able to enter how big the board is and what ships they will play with.

Comment (BrianZanti) [reply module1/projects/battleship/requirements.md]: Or, add some sort of "smart" computer

Comment (BrianZanti) [reply module1/projects/battleship/requirements.md]: Or, make the game two player

**Comment by memcmahon** [comment]
Updates made!

**Pull request closed without merging**